docs/requirements.txt

```diff
@@ -25,3 +25,7 @@ huggingface_hub==0.7
 torchaudio==0.11.0
 fasttext==0.9.2
 timm==0.6.5
+numpy==1.21.6
```

Comment:
Let's remove numpy and any other dependencies of cleanlab as well.

This file is only supposed to list extra dependencies which aren't installed when cleanlab is installed